src/app/parcels/add/formSections/CollectionDateCard.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { CardProps, onChangeDateOrTime, errorText } from "@/components/Form/formFunctions";
+import GenericFormCard from "@/components/Form/GenericFormCard";
+import StyledDatePicker from "@/components/DataInput/DatePicker";
+import { ErrorText } from "@/components/Form/formStyling";
+
+const CollectionDateCard: React.FC<CardProps> = ({
+    fieldSetter,
+    errorSetter,
+    formErrors,
+    fields,
+}) => {
+    return (
+        <GenericFormCard
+            title="Collection Date"
+            required={true}
+            text="What date is the client collecting their parcel?"
+        >
+            <>
+                <StyledDatePicker
```

Comment:
Is it possible to not allow users to select date which already passed? Might not be desirable idk.

Reply:
The bar now stays red for if the day has already passed. Don't think we want to change it fully incase retrospective dates need to be inputted, but it gives some warning at least!